CMakeLists.txt

```diff
@@ -0,0 +1,213 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
We are going to be using catkin_simple in all our ROS packages. Please follow the template of catkin_simple CMakeLists.txt